From 4f4423e1198b3b76c2dfa2820c2ccb3345df4105 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dsteve@broadcom.com>
Date: Fri, 14 Mar 2014 17:02:45 +0000
Subject: [PATCH 178/196] V4L2: Increase the number of image buffers for high
 fps

Increase the number of image buffers available to improve
performance at high framerates.

Signed-off-by: Dave Stevenson <dsteve@broadcom.com>
---
 drivers/media/platform/bcm2835/bcm2835-camera.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/bcm2835/bcm2835-camera.c b/drivers/media/platform/bcm2835/bcm2835-camera.c
index 3dea993..b6880cc 100644
--- a/drivers/media/platform/bcm2835/bcm2835-camera.c
+++ b/drivers/media/platform/bcm2835/bcm2835-camera.c
@@ -1330,7 +1330,7 @@ static int set_camera_parameters(struct vchiq_mmal_instance *instance,
 		.one_shot_stills = 1,
 		.max_preview_video_w = 1920,
 		.max_preview_video_h = 1088,
-		.num_preview_video_frames = 3,
+		.num_preview_video_frames = 6,
 		.stills_capture_circular_buffer_height = 0,
 		.fast_preview_resume = 0,
 		.use_stc_timestamp = MMAL_PARAM_TIMESTAMP_MODE_RAW_STC
-- 
1.9.1

